airflow/jobs/scheduler_job_runner.py

```diff
@@ -1805,6 +1815,7 @@ def _fail_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
             )
         ).all()
 
+        num_allowed_retries = conf.getint("core", "num_stuck_retries", fallback=2)
```

Comment:
```suggestion
        num_allowed_retries = conf.getint("core", "num_stuck_retries")
```

You need to add this to [config.yml](https://github.com/apache/airflow/blob/main/airflow/config_templates/config.yml) (and then won't need a fallback).